client/verta/verta/operations/monitoring/summaries.py

```diff
@@ -117,6 +153,23 @@ def __repr__(self):
             self._ids, self._names, self._type_names, self._monitored_entity_ids
         )
 
+    def __add__(self, other):
+        assert isinstance(other, self.__class__)
```

Comment:
Returning `NotImplemented` à la [the data model reference](https://docs.python.org/3/reference/datamodel.html#emulating-numeric-types).

Using `type(self)` instead of `self.__class__` because I'd personally favor using more public functions than dunders.
```suggestion
        if not isinstance(other, type(self)):
            return NotImplemented
```